app/controllers/spree/admin/users_controller.rb

```diff
@@ -127,9 +129,13 @@ def new_email_unconfirmed?
         params[:user][:email] != @user.email
       end
 
+      def build_resource
+        model_class.new(locale: spree_current_user.locale)
+      end
+
       def user_params
         ::PermittedAttributes::User.new(params).call(
-          %i[enterprise_limit show_api_key_view]
+          %i[enterprise_limit locale show_api_key_view]
```

Comment:
This parameter can be permitted in general in `PermittedAttributes::User` because the user is allowed to change their own language.

Reply:
Applied in 07353de7870a5321ab26d30df57e1861b67ffe86